en/docs/guides/Provisioning/outbound-provisioning-with-google.md

```diff
@@ -0,0 +1,148 @@
+    1.  Go to your domain's admin console via https://admin.google.com.
+    2.  Click **Security**.
+
+        ![admin-console-security](../assets/img/guides/admin-console-security.png) 
+
+    3.  Click **Advanced settings \> Manage API client access**.
+    4.  Fill the following values:
+        1.  Paste the [Client ID value you copied previously](../../guides/outbound-provisioning-with-google#copy)
+            as the value for Client Name.
+        2.  Enter
+            `                             https://www.googleapis.com/auth/admin.directory.user,https://www.googleapis.com/auth/admin.directory.orgunit,https://www.googleapis.com/auth/admin.directory.group                           `
+            as the value for scopes.
+        3.  Click **Authorize**.
+
+        ![manage-api-client-access](../assets/img/guides/manage-api-client.png) 
+
+7.  Enable Admin SDK.
+    1.  On the Open the [Google developers
+        console](https://console.developers.google.com/cloud-resource-manager)
+       , click the menu icon, and click **APIs & Services** > **Dashboards**.
+    2.  Click on **Enable APIs AND Services**.  
+    3.  Search for Admin SDK and click **Enable**.
+
+        ![admin-sdk](../assets/img/guides/admin-sdk.png)
+
+## Configuring the WSO2 Identity Server to use email address as the username
+
+Provisioning is the process of coordinating the creation of user
+accounts, e-mail authorizations in the form of rules and roles, and
+other tasks such as provisioning of resources associated with enabling
+new users.
+
+1.  Download the WSO2 Identity Server from
+    [here](http://wso2.com/products/identity-server/).
+2.  <a name= "email"></a>When you log into Google, you normally use an email address. So, to
+    integrate this with the Identity Server, you need to configure WSO2
+    IS to enable users to log in using their email addresses. In order
+    to do that, follow the steps found in [Using Email Address as
+    the
+    Username](insert-link)
+    topic.
+3.  Restart the Identity Server. Since the username and password of the
+    admin user were updated, start the WSO2 IS server using the -Dsetup
+    parameter as shown in the command below.
+
+    ``` curl
+    sh wso2server.sh -Dsetup
+    ```
+
+Now that you are done with configuring WSO2 Identity Server to use the
+email address, configure the identity provider and the service provider.
+
+## Configure Google as the identity provider
+
+(insert-admin-portal-content)
+
+## Configure WSO2 IS as the resident service provider
+
+(insert-admin-portal-content)
+
+## Manage users
+
+(insert-admin-portal-content)
+
+## What's next?
+
+-   If you want to JIT provision users from Google to WSO2 Identity
+    Server in this use case, see [Configuring Just-In-Time Provisioning
+    for an Identity
+    Provider](insert-link)
+.
+-   You can configure WSO2 IS to outbound provision users only if a
+    given XACML rule is met. For more information, see [Rule-Based
```

Comment:
"only if a given XACML rule is set" --> this implies that oubound provisioning of users can only happen if we set a XACML rule. I don't think that's true..Please check if  we want to say this instead:

"You can configure WSO2 IS to outbound provision users according to a given XACML rule"